libvast/src/system/application.cpp

```diff
@@ -357,9 +357,10 @@ auto make_spawn_command() {
 }
 
 auto make_status_command() {
-  return std::make_unique<command>("status",
-                                   "shows various properties of a topology",
-                                   documentation::vast_status, opts());
+  return std::make_unique<command>(
+    "status", "shows various properties of a topology",
+    documentation::vast_status,
+    opts().add<caf::atom_value>("verbosity,v", "output verbosity level"));
```

Comment:
Shouldn't this just re-use our global `system.verbosity` setting?

If you want to keep them separate, please introduce a different name to avoid confusion, and don't re-use the same short option.

Reply:
If we separate the verbosity from the object it attaches to, then `system.verbosity` and `status.verbosity` are semantically the same. While we could rename this option to `--detail` it seems somewhat arbitrary to me.

Reusing `system.verbosity` would require the user to write `vast -v debug status` instead of `vast status -v debug`.

I kept this consistent with the log level option intentionally. But we could also change it to `--verbose` and `--debug` flags.

Reply:
From a user perspective, I find having options that are named exactly the same do different things depending on their position and take different arguments quite confusing.

Reply:
In this case I'm agreeing with @tobim. I would find a confusing that I have to control the output (stdout) through the same option that controls a different channel (stderr).

Reply:
In that case I'd still like to see a different long and short option. Using `verbose,v` for both _will_ confuse users. 

What do you think about `detail`?

Reply:
It was my understanding the we have 3 levels, just like with logging: info -> verbose -> debug. So using `-v`, `-vv` and `-vvv` seems easy to remember. (Just to illustrate, I know we don't support multiple options of the same type.)

I'm not against using something like `detail` as a prefix for the different output levels.

Reply:
You're misunderstanding me @mavam. I don't want these options to have the same name, as that conflicts with a future refactoring we have planned for the CLI for allowing options to appear at a later point than the subcommand they relate to. If we are not merging the `system.verbosity` and `status.verbosity` options, we should _rename_ the `status.verbosity` option, e.g., to `status.detail`.

I'd like to see the short option removed altogether. This command is not used a lot, and we should not re-use short options.

Reply:
I'm fine with using clearer naming that differentiates from the logging nomenclature. My only concern is that it should be clear that we multiple levels of detail - 3 in fact. So call them `--detail`, `--meticulous`, `--comprehensive`, or what you like. The max level could be `--developer` to make it clear that it has debug-like semantics.